src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
```

Comment:
for folder structure, inteface could be here, but for implementations, create a sepsrate impl folder